package.json

```diff
@@ -116,7 +117,8 @@
     "stream-throttle": "^0.1.3",
     "terser-webpack-plugin": "^4.2.3",
     "typescript": "^4.8.4",
-    "url-search-params": "^0.10.0"
+    "url-search-params": "^0.10.0",
+    "whatwg-fetch": "^3.6.2"
```

Comment:
nts: should also make sure `whatwg-fetch` makes it into `dependencies`, and consider peer dependencies for polyfills where applicable